src/tagsouppullparser.cpp

```diff
@@ -68,12 +68,14 @@ tagsouppullparser::event tagsouppullparser::next() {
 		skip_whitespace();
 		if (inputstream->eof()) {
 			current_event = event::END_DOCUMENT;
-			break;
 		}
-		if (c != '<') {
+		else if (c != '<') {
```

Comment:
Can you make this `} else if (…) {`? We mostly use this style throughout Newsbeuter, though some sections do put `else if` on a separate line.